diff --git a/globals.d.ts b/globals.d.ts
index 20d17e083ca2fbf86dbe55e61ee741605bbe2491..9ed119d607b9d9fcd5c96c39f2779f82fb718557 100644
--- a/globals.d.ts
+++ b/globals.d.ts
@@ -1008,7 +1008,7 @@ interface ArrayBuffer {
    * Returns a section of an ArrayBuffer.
    */
   slice(begin: number, end?: number): ArrayBuffer;
-  readonly [Symbol.toStringTag]: string;
+  readonly [Symbol.toStringTag]: "ArrayBuffer";
 }
 
 interface SharedArrayBuffer {
@@ -1284,7 +1284,7 @@ interface ImportMeta {
    * )
    * ```
    */
-  readonly main: boolean;
+  main: boolean;
 
   /** Alias of `import.meta.dir`. Exists for Node.js compatibility */
   dirname: string;
